variables.env

```diff
@@ -0,0 +1,10 @@
+NODE_ENV=development
```

Comment:
Do not please store such data in environment variables, it's not secure. Just create file with constants